analyzers/src/ITs.JsonParser/Json/RuleIssues.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace ITs.JsonParser.Json;
+
+// Format of JSONs in "actual/" and "expected/"
+internal class RuleIssues
+{
+    public RuleIssue[] Issues { get; set; }
+}
+
+internal class RuleIssue
+{
+    public string Id { get; set; }
+    public string Message { get; set; }
+    public Location Location { get; set; }
```

Comment:
The change to the flat format will be a different PR, right?

Reply:
Yes, [this one](https://github.com/SonarSource/sonar-dotnet/issues/8649)